example/myInterface.go

```diff
@@ -22,6 +25,7 @@ type MyInterface interface {
 	NamedParam(str string)
 	NamedVariadicParam(strs ...string)
 	SameTypeNamedParams(str1, str2 string)
+	InternalTypeParam(internal internal.Internal)
```

Comment:
While I was here, I added another method to check how `mock` handles local/internal types.